greatNovels.sql

```diff
@@ -0,0 +1,149 @@
+create database greatNovels;
+
+-- create user 'greatNovels'@'localhost' identified with mysql_native_password by 'b00kw0rm';
```

Comment:
With this line commented out when I run your SQL you'd have connection errors and none of your routes would work. Be careful with what you check in.